tests/math_builtin_api/math_builtin.h

```diff
@@ -102,15 +102,15 @@ verify(sycl_cts::util::logger &log, T value, sycl_cts::resultRef<T> r, int accur
   log.note("value: " + printable(value) + ", reference: " +
            printable(reference));
   std::string msg = "Expected accuracy in ULP: " + std::to_string(accuracy);
-  if (comment != "")
-    msg += ", " + comment;
+  if (!comment.empty()) msg += ", " + comment;
   log.note(msg);
   return false;
 }
 
 template <typename T>
-typename std::enable_if<std::is_integral<T>::value, bool>::type
-verify(sycl_cts::util::logger &log, T value, sycl_cts::resultRef<T> r, int, std::string) {
+typename std::enable_if<std::is_integral<T>::value, bool>::type verify(
```

Comment:
Since you are here :-)
```suggestion
typename std::enable_if_t<std::is_integral_v<T>, bool> verify(
```